lib/utils/httputil/httputil.go

```diff
@@ -295,16 +308,9 @@ func Send(method, rawurl string, options ...SendOption) (resp *http.Response, er
 		}
 	}
 
-	interval := opts.retry.interval
-	for i := 0; i < opts.retry.max; i++ {
-		if i > 0 {
-			time.Sleep(interval)
-			interval = min(
-				time.Duration(float64(interval)*opts.retry.backoffMultiplier),
-				opts.retry.backoffMax)
-		}
+	var resp *http.Response
+	for {
 		resp, err = client.Do(req)
-
 		httpFallbackDisabled := opts.httpFallbackDisabled
 		if !httpFallbackDisabled && is5xxResponse(resp) {
```

Comment:
I think `is5xxResponse(resp)` is not needed here because the logic is moved to line 334